mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -111,6 +112,7 @@ public FakeMailContext build() {
             private MimeMessage msg;
             private Map<String, Serializable> attributes = new HashMap<String, Serializable>();
             private Optional<String> state = Optional.absent();
+            private boolean fromMailet = false;
```

Comment:
1. The habit, and consistence with the code
2. What if I want to add a .notSentFromAMailet() later to make some of my future tests more explicit ?